numerics/elliptic_integrals.cpp

```diff
@@ -60,7 +60,7 @@ void Elbdj(double const phi,
   // TODO(phl): CHECK_EQ(π / 2, phi + phic);
   double m, nc, h, c, x, d2, z, bc, dc, jc, sz, t, v, t2;
 
-  if (phi < 1.345) {
+  if (phi < 1.249) {
```

Comment:
Can we document this?
Previously the lack of documentation was explained by "translated from undocumented FORTRAN", but if we start making changes we should probably comment our magic constants.

Reply:
I added a big bad block of comments, but mostly this is "let's make the code do what the paper says" (the paper has like 2 pages to explain this constant).